appveyor.yml

```diff
@@ -14,16 +14,19 @@ init:
 
 before_build:
   - ps: .\build\Set-BuildVersion -Version $env:APPVEYOR_BUILD_VERSION -Suffix $env:VERSION_SUFFIX
-  - appveyor-retry dotnet restore -v Minimal
+  - appveyor-retry dotnet restore -v Minimal  
 
 build_script:
-  - ps: dotnet build -c Release
+  - ps: dotnet build -c Release 
 
+test_script:

```

Comment:
Did you come across an issue with unit tests? They should be picked up automatically via appveyor. The test harness is a shared library that the other foundatio libs build on, I believe (@ejsmith can confirm) that only the Foundation.Tests are the only runable tests?